app/_src/kubernetes-ingress-controller/references/cli-arguments.md

```diff
@@ -53,44 +53,57 @@ Following table describes all the flags that are available:
 | `--log-format`                           | `string`           | Format of logs of the controller. Allowed values are text and json.                                                                                 | `"text"`                          |
 | `--log-level`                            | `string`           | Level of logging for the controller. Allowed values are trace, debug, info, warn, error, fatal and panic.                                           | `"info"`                          |
 | `--metrics-bind-address`                 | `string`           | The address the metric endpoint binds to.                                                                                                           | `":10255"`                        |
-| `--profiling`                            | `boolean`          | Enable profiling via web interface `host:10256/debug/pprof/`.                                                                                          | `false`                           |
+| `--profiling`                            | `boolean`          | Enable profiling via web interface `host:10256/debug/pprof/`.                                                                                       | `false`                           |
 | `--proxy-sync-seconds`                   | `float32`          | Define the rate (in seconds) in which configuration updates will be applied to the Kong Admin API.                                                  | `3`                               |
 | `--proxy-timeout-seconds`                | `float32`          | Define the rate (in seconds) in which the timeout configuration will be applied to the Kong client.                                                 | `10`                              |
 | `--publish-service`                      | `string`           | Service fronting Ingress resources in "namespace/name" format. The controller will update Ingress status information with this Service's endpoints. |                                   |
 | `--publish-status-address`               | `strings`          | User-provided addresses in comma-separated string format, for use in lieu of "publish-service" when that Service lacks useful address information.  |                                   |
 
-{% if_version gte: 2.4.x %}
+{% if_version gte:2.9.x %}
+| `--publish-service-udp`                  | `string`           | Service fronting UDP routing resources in "namespace/name" format. The controller will update Ingress status information with this Service's endpoints. If omitted, the same Service will be used for both TCP and UDP routes. |                                   |
+| `--publish-status-address-udp`           | `strings`          | User-provided addresses in comma-separated string format, for use in lieu of "publish-service-udp" when that Service lacks useful address information. |                                   |
```

Comment:
```suggestion
| `--publish-status-address-udp`           | `strings`          | User-provided addresses in comma-separated string format, for use instead of `publish-service-udp` when that Service lacks useful address information. |                                   |
```
Switched `in lieu of` to `instead of` for simplicity + clarity for non-native speakers.